tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -89,7 +82,7 @@ def test_do_not_inhibit_if_module_is_force_loaded(shell, convert2rhel):
     Force loaded kmods are denoted (FE) where F = module was force loaded E = unsigned module was loaded.
     Convert2RHEL sees force loaded kmod as tainted.
     """
-    if "oracle-7" not in system_version and "centos-7" not in system_version:
+    if SYSTEM_RELEASE in ("oracle-7", "centos-7"):
```

Comment:
This should be 
```suggestion
    if SYSTEM_RELEASE not in ("oracle-7", "centos-7"):
```

Right? The old statement is checking against the `centos-7` and `oracle-7`